src/modules/airspeed_selector/airspeed_selector_params.c

```diff
@@ -239,3 +240,18 @@ PARAM_DEFINE_INT32(ASPD_FS_T_START, -1);
  * @group Airspeed Validator
  */
 PARAM_DEFINE_FLOAT(ASPD_WERR_THR, 0.55f);
+
+/**
+ * First principle airspeed check time window
+ *
+ * Time window for comparing airspeed change to throttle and pitch change.
+ * If the airspeed change within this window is negative while throttle increases
+ * and the vehicle pitches down, the check will trigger.
+ * This check is meant to catch degrading airspeed blockages as can happen when flying through icing conditions.
```

Comment:
I would add a comment that FW_THR_TRIM has to be set precisely otherwise it will not work well.